src/normal.cc

```diff
@@ -553,6 +553,8 @@ void pipe(Context& context, NormalParams)
                 ScopedEdition edition(context);
                 ForwardChangesTracker changes_tracker;
                 size_t timestamp = buffer.timestamp();
+                Vector<StringView> inputs;
```

Comment:
storing StringViews will fail here, as the String they are referencing dies, use a String, and std::move(in) into it.

Reply:
Thank you, this fixed the issue